### PR TITLE
OSDOCS-4745: Installing Nutanix cluster in a restricted env

### PR DESCRIPTION
**Pull request by mjpytlak**
Version(s):
4.12+

Issue:
This PR addresses [osdocs-4745](https://issues.redhat.com/browse/OSDOCS-4745).

Link to docs preview:
[Installing a cluster on Nutanix in a restricted network](https://54259--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_nutanix/installing-restricted-networks-nutanix-installer-provisioned.html)

A fair amount of the content in this assembly is reused from existing approved Nutanix material. The following preview links are for areas where new content was added in support of a restricted installation:

- [Supported installation methods for different platforms](https://54259--docspreview.netlify.app/openshift-enterprise/latest/installing/installing-preparing.html#supported-installation-methods-for-different-platforms) (Update to support matrix indicating support for a restricted installation)
- [Image set configuration examples](https://54259--docspreview.netlify.app/openshift-enterprise/latest/installing/disconnected_install/installing-mirroring-disconnected.html#oc-mirror-image-set-examples_installing-mirroring-disconnected) > Use case: Including the Nutanix CSI Operator (final use case in list)
- [Downloading the RHCOS cluster image](https://54259--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_nutanix/installing-restricted-networks-nutanix-installer-provisioned.html#installation-nutanix-download-rhcos_installing-restricted-networks-nutanix-installer-provisioned)
- [Sample customized install-config.yaml file for Nutanix](https://54259--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_nutanix/installing-restricted-networks-nutanix-installer-provisioned.html#installation-nutanix-config-yaml_installing-restricted-networks-nutanix-installer-provisioned)
- [Disabling the default OperatorHub catalog resources](https://54259--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_nutanix/installing-restricted-networks-nutanix-installer-provisioned.html#olm-restricted-networks-operatorhub_installing-restricted-networks-nutanix-installer-provisioned)
- [Installing the policy resources into the cluster](https://54259--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_nutanix/installing-restricted-networks-nutanix-installer-provisioned.html#oc-mirror-updating-cluster-manifests_installing-restricted-networks-nutanix-installer-provisioned)

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

**Comment by mjpytlak** [comment]
@vnephologist @sgaoshang  Draft PR is ready for review. PTAL. Thank you.

**Comment by vnephologist** [comment]
cc: @yannickstruyf3

**Comment by yannickstruyf3** [comment]
LGTM

**Comment by mjpytlak** [comment]
@sgaoshang Following up on my request for QE review. PTAL. Thank you in advance.

**Comment by sgaoshang** [comment]
Currently we still have issue with restricted env during cluster installation, will update after it resolved, thanks.

**Comment by sgaoshang** [comment]
@mjpytlak restricted installation succeed, /lgtm, thanks.

File: modules/cli-installing-cli.adoc [hunk 1]
```diff
@@ -63,9 +65,7 @@ command-line interface. You can install `oc` on Linux, Windows, or macOS.
 
 [IMPORTANT]
 ====
-If you installed an earlier version of `oc`, you cannot use it to complete all
-of the commands in {product-title} {product-version}. Download and
-install the new version of `oc`.
+If you installed an earlier version of `oc`, you cannot use it to complete all of the commands in {product-title} {product-version}. Download and install the new version of `oc`.
```
Comment (sheriff-rh) [inline modules/cli-installing-cli.adoc:68]: I know this is intentionally vague, but is there a specific release that works? Can you say oc v1.0+, for example?

File: modules/registry-configuring-storage-nutanix.adoc [hunk 2]
```diff
@@ -0,0 +1,9 @@
+// * installing/installing-restricted-networks-nutanix-installer-provisioned.adoc
+
+:_content-type: PROCEDURE
+[id="registry-configuring-storage-nutanix_{context}"]
+= Configuring the default storage container
+
+After you install the cluster, you must install the Nutanix CSI Operator and configure the default storage container for the cluster.
```
Comment (sheriff-rh) [inline modules/registry-configuring-storage-nutanix.adoc:7]: Is the Nutanix Operator supported by Red Hat? If not, do we need to include a note block stating this?

**Review by sheriff-rh — COMMENTED** [review]
Excellent work, Mike. This is complex work and you've done a great job of keeping it organized. It presents and reads well! 

I have two questions in my review.

File: modules/registry-configuring-storage-nutanix.adoc (hunk 2 above)
Comment (mjpytlak) [reply modules/registry-configuring-storage-nutanix.adoc:7]: Yes it is supported, but must be installed separately post-installation. For the time being the guidance from the team was to simply xref the Nutanix doc.

File: modules/cli-installing-cli.adoc (hunk 1 above)
Comment (mjpytlak) [reply modules/cli-installing-cli.adoc:68]: TBH. I cannot answer. I do not own this module. I simply reuse it in the context of installation assemblies.

**Comment by wolfganghuse** [comment]
LGTM

**Review by makentenza — APPROVED** [review]
LGTM

**Comment by sjstout** [comment]
LGTM

**Pull request merged**